tests/asteroid_agent_test.py

```diff
@@ -85,3 +92,35 @@ def testCheck_whenVersionNotVulnerable_shouldNotReportVulnerability() -> None:
     with mock.patch.object(exploit, "accept", return_value=False):
         vulnerabilities = exploit.check(target)
         assert len(vulnerabilities) == 0
+
+
+def testAgent_whenCustomCVEPassed_shouldScanOnlyTheScope(
+    agent_mock: list[message.Message],
+) -> None:
+    """Fixture of the Nmap Agent to be used for testing purposes."""
+    with (pathlib.Path(__file__).parent.parent / "ostorlab.yaml").open() as yaml_o:
+        definition = agent_definitions.AgentDefinition.from_yaml(yaml_o)
+        settings = runtime_definitions.AgentSettings(
+            key="agent/ostorlab/nmap_agent",
+            bus_url="NA",
+            bus_exchange_topic="NA",
+            args=[
+                utils_definitions.Arg(
+                    name="custom_CVE_list",
+                    type="array",
+                    value=json.dumps(
+                        [
+                            "FastCGIIntegerOverflowExploit",
+                            "CalderaAgentCompilationRCEExploit",
```

Comment:
These should be CVE. You should fix the plugin to have as a key CVE.

Reply:
It is the case, but there are some exploits without CVE. Change the unit test to use the one with CVE in the key!